homefork fo MyFirstApp2 lection

katkalof opened the pull request.

katkalof committed:
add AboutFragment for showing App's info and send email to me

rgrigoryev commented on app/src/main/java/ru/katkalov/android/yamobdev2016/ui/activity/MainActivity.java:
```diff
@@ -42,9 +49,20 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 }
                 return true;
+            case R.id.menu_about:
+                getFragmentManager()
```
А ты используешь не android.support Fragment'ы?
Не надо пока переделывать - интересно убедят ли тебя использовать именно их )

rgrigoryev commented on app/src/main/java/ru/katkalov/android/yamobdev2016/ui/fragment/AboutFragment.java:
```diff
@@ -0,0 +1,43 @@
+            case R.id.send_email_to_developer:
+                Intent i = new Intent(Intent.ACTION_SEND);
+                i.setType("plain/text");
+                i.putExtra(Intent.EXTRA_EMAIL, getResources().getStringArray(R.array.emails));
```
https://youtu.be/b6zKBZcg5fk?t=15m6s

rgrigoryev commented on app/src/main/java/ru/katkalov/android/yamobdev2016/ui/fragment/AboutFragment.java:
```diff
@@ -0,0 +1,43 @@
+        switch (view.getId()) {
+            case R.id.send_email_to_developer:
+                Intent i = new Intent(Intent.ACTION_SEND);
+                i.setType("plain/text");
```
Тут предложатся не только почтовые приложения. Даже Диск предложится. Хочется именно письмо.

rgrigoryev commented on app/src/main/java/ru/katkalov/android/yamobdev2016/ui/fragment/ArtistFragment.java:
```diff
@@ -68,21 +68,14 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
+        super.onDestroy();
         mArtist = null;
         if ( mPicasso != null ) {
             mPicasso.shutdown();
```
Не надо глушить его так часто https://github.com/square/picasso/issues/321#issuecomment-29920131

katkalof committed:
Fixed
"Тут предложатся не только почтовые приложения. Даже Диск предложится. Хочется именно письмо."

katkalof committed:
Fixed https://github.com/katkalof/YaMobDev2016/pull/1#discussion_r71369098

"Санитарная оптимизация" ©

katkalof committed:
add HeadsetPlug-check feature.
it's work begins by booting device becase added bootingreceiver and service, which enable HeadsetPlug checking

katkalof committed:
change Toast to normal notification about HeadsetPlug

katkalof committed:
add WORKING proguard finally
no jack and jill more! no god please no
fix bug with AboutFragment
